Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -284,6 +285,22 @@ def test_is_feed(self, yaml_context, yaml_type, is_actually_feed):
         dummy_pack.is_feed_pack(yaml_context, yaml_type)
         assert dummy_pack.is_feed == is_actually_feed
 
+    def test_remove_unwanted_files(self):
+        """
+           Given:
+               - Pack name & path.
+           When:
+               - Before uploading a pack to the marketplace, delete unwanted files.
+           Then:
+               - Assert the TestPlaybooks die was deleted.
+       """
+        os.mkdir('Tests/Marketplace/Tests/test_data/pack_to_test')
+        os.mkdir('Tests/Marketplace/Tests/test_data/pack_to_test/TestPlaybooks')
+        test_pack = Pack(pack_name="pack_to_test", pack_path='Tests/Marketplace/Tests/test_data/pack_to_test')
+        test_pack.remove_unwanted_files()
+        assert not os.path.isdir('Tests/Marketplace/Tests/test_data/pack_to_test/TestPlaybooks')
+        shutil.rmtree('Tests/Marketplace/Tests/test_data/pack_to_test')
```

Comment:
Let's add here another folder like "integrations" and check that it was not deleted and also another folder inside the TestPlaybooks (like we had that deprecated folder)